service/src/main/java/org/ehrbase/service/FhirTerminologyServerR4AdaptorImpl.java

```diff
@@ -147,6 +147,7 @@ public final DvCodedText lookUp(final String conceptId) {
 	@Override
 	public final Boolean validate(final DvCodedText concept, final String valueSetId) {
 		// TODO Auto-generated method stub
+		System.out.println("inside the validate method of R4 implemetntation");
```

Comment:
I would suggest to use a logger and use a log.debug(...) instead

Reply:
As Luis is absent for some time, could you please do the changed and merge?

Reply:
Done